### PR TITLE
feat: saliency map

### PR DESCRIPTION
**Pull request by vivek-mandal**
Test cases added for Saliency Maps

### CODE REVIEW

File: tutorials/saliencyMaps/UDF.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+class MRICNN(PytorchAbstractClassifierUDF):
+
+    @property
+    def name(self) -> str:
+        return "MRICNN"
+
+    def setup(self):
+        # !wget -nc "https://www.dropbox.com/s/cnsgyitrtw40lgs/model.pth?dl=0"
```
Comment (jarulraj) [inline tutorials/saliencyMaps/UDF.py:28]: We can get rid of these comments.

File: tutorials/saliencyMaps/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+evadb
```
Comment (jarulraj) [inline tutorials/saliencyMaps/requirements.txt:1]: Don't need pydicom

Comment (jarulraj) [reply tutorials/saliencyMaps/requirements.txt:1]: In fact, don't need this file itself

File: test/integration_tests/test_udf.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        select_query_saliency = """SELECT data, MRICNN(data)
+                        FROM MRI;
+                       """
+        actual_batch_saliency = execute_query_fetch_all(select_query_saliency)
+        print(actual_batch_saliency.frames)
+        
+        df = actual_batch_saliency.frames
+        fig, ax = plt.subplots(nrows=2, ncols=len(df), figsize=[15,18])
+
+        for i in range(len(df)):
```
Comment (jarulraj) [inline test/integration_tests/test_udf.py:71]: Remove plotting code. Programmatically check the size of the batch similar to feature vector tests.

File: test/integration_tests/test_udf.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import pandas as pd
+import pytest
+
+from eva.catalog.catalog_manager import CatalogManager
+from eva.models.storage.batch import Batch
+from eva.server.command_handler import execute_query_fetch_all
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+# mpl.use('TkAgg')
```
Comment (jarulraj) [inline test/integration_tests/test_udf.py:33]: Remove all plotting code.

**Comment by jarulraj** [comment]
Look at the failing tests here -- https://app.circleci.com/pipelines/github/georgia-tech-db/eva/3507/workflows/1e997811-781d-47d2-9a0b-4ff47b13ed6f/jobs/15920

**Comment by jarulraj** [comment]
Locally run the test script -- https://evadb.readthedocs.io/en/stable/source/contribute/index.html#submitting-a-contribution

**Review by jarulraj — CHANGES_REQUESTED** [review]

File: tutorials/saliencyMaps/requirements.txt (hunk 2 above)
Comment (vivek-mandal) [reply tutorials/saliencyMaps/requirements.txt:1]: Done

**Comment by vivek-mandal** [comment]
@jarulraj You can get a copy of the model to from the below link https://www.dropbox.com/s/cnsgyitrtw40lgs/model.pth?dl=0
You can test it locally, this is working from end

File: test/integration_tests/test_udf.py (hunk 3 above)
Comment (vivek-mandal) [reply test/integration_tests/test_udf.py]: Hello @jarulraj 
For checking the size we would need two entities, the first will be the length of our table created from the saliency udf, i am unsure about the second entity, can you guide me here

**Comment by jarulraj** [comment]
Subsumed in  #714

**Pull request closed without merging**